### PR TITLE
fix git hook

### PR DESCRIPTION
**Pull request by fffw**

### CODE REVIEW

File: git-hook [hunk 1]
```diff
@@ -27,16 +27,16 @@ which errcheck >/dev/null || (echo "Unable to find errcheck, please install it:
 which golint >/dev/null || (echo "Unable to find golint, please install it: \`go get -u github.com/golang/lint/golint\`" && exit 1)
 
 echo "*** Running \033[0;34mErrcheck\033[0m ***" && \
-for dir in *; do
+for dir in $MODIFIED_DIRS; do
   errcheck github.com/getlantern/$dir || \
-    echo "\033[0;31merrcheck returned error analyzing the package '$dir'\033[0m" && \
+    (echo "\033[0;31merrcheck returned error analyzing the package '$dir'\033[0m" && \
```
Comment (fffw) [inline git-hook]: Explicitly group statements, as in Bash,  || and && are the same precedence.

File: git-hook [hunk 2]
```diff
@@ -27,16 +27,16 @@ which errcheck >/dev/null || (echo "Unable to find errcheck, please install it:
 which golint >/dev/null || (echo "Unable to find golint, please install it: \`go get -u github.com/golang/lint/golint\`" && exit 1)
 
 echo "*** Running \033[0;34mErrcheck\033[0m ***" && \
-for dir in *; do
+for dir in $MODIFIED_DIRS; do
```
Comment (fffw) [inline git-hook]: Only check modified directions.

**Comment by fffw** [comment]
git hook is super useful to catch errors. Thanks @uaalto ! It would be good if you can take a look at this quick fix.

**Comment by fffw** [comment]
Oh I just noticed that it can only act as pre-commit hook after my change, as when `git push`, staging area is clean. But it will take too long to errcheck and test race for all code, pre-commit hook with only the changed directory might be more useful.

File: git-hook (hunk 2 above)
Comment (uaalto) [inline git-hook]: wow, this was definitely meant to be! Thank you @fffw !

File: git-hook (hunk 1 above)
Comment (uaalto) [inline git-hook]: Thank you! A very tiny detail, though. Could you please place the `(` as in the other cases? (or change the other cases if you prefer). Even though is a tiny style detail, Bash is already a quirky language, so uniformity pays off in my opinion.

Comment (fffw) [inline git-hook]: Strange enough I didn't notice the brackets below! Will correct.

**Comment by fffw** [comment]
Updated @uaalto .

**Comment by uaalto** [comment]
Awesome! LGTM.

**Pull request merged**